airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/pull_request/commands.py

```diff
@@ -69,9 +69,26 @@
     required=False,
     default=False,
 )
+@click.option(
```

Comment:
💅 Instead of an `auto-merge` flag we could decide to expose a `--label` option set to `multiple` which could offer a more generic interface for other use case.